### PR TITLE
Update GuidelinesForNewRules.md

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

File: GuidelinesForNewRules.md [hunk 1]
```diff
@@ -25,25 +25,14 @@
...
+   Please review [Contribute docs for .NET code analysis rules to the .NET docs repository
+](https://docs.microsoft.com/contribute/dotnet/dotnet-contribute-code-analysis) for guidelines.
 
-      `TIP:` Clone an existing `caxxxx.md` file inside `/docs/code-quality` sub-folder in the repo, rename it and update the contents for the new rule.
-   2. Update the following tables in the repo for supported CA rule IDs:
-      1. Add entry in `/docs/code-quality/toc.yml` under appropriate category.
-      2. Add entry in `/docs/code-quality/code-analysis-warnings-for-managed-code-by-checkid.md`
-      3. Add entry in the documentation file `/docs/code-quality/<%category%>-warnings.md` for rule's `Category`. For example:
-         1. For a new rule with category `Design`, add an entry to `/docs/code-quality/design-warnings.md`.
-         2. For a new rule with category `Performance`, add an entry to `/docs/code-quality/performance-warnings.md`, and so on.
```
Comment (Youssef1313) [inline GuidelinesForNewRules.md:47]: These are a little bit outdated, and the parallel information already exist in [Contribute docs for .NET code analysis rules to the .NET docs repository](https://docs.microsoft.com/contribute/dotnet/dotnet-contribute-code-analysis).

File: GuidelinesForNewRules.md [hunk 2]
```diff
@@ -25,25 +25,14 @@
...
-
-If for some exceptional reason you are unable to submit a PR, please [file a documentation issue](https://github.com/MicrosoftDocs/visualstudio-docs/issues) to add documentation for the rule in future. Please include all relevant information in the issue to allow the documentation experts to easily author the documentation. For example, see [this issue](https://github.com/MicrosoftDocs/visualstudio-docs/issues/3454).
+If for some exceptional reason you are unable to submit a PR, please [file a documentation issue](https://github.com/dotnet/docs/issues) to add documentation for the rule in future. Please include all relevant information in the issue to allow the documentation experts to easily author the documentation. For example, see [this issue](https://github.com/MicrosoftDocs/visualstudio-docs/issues/3454).
```
Comment (Youssef1313) [inline GuidelinesForNewRules.md:38]: Should we keep the example issue given that it's in visualstudio-docs?

Comment (mavasani) [reply GuidelinesForNewRules.md:38]: Yeah, I think we can remove the example issue.

**Review by mavasani — APPROVED** [review]
Thanks @Youssef1313!

File: GuidelinesForNewRules.md (hunk 2 above)
Comment (Youssef1313) [reply GuidelinesForNewRules.md:38]:
```suggestion
If for some exceptional reason you are unable to submit a PR, please [file a documentation issue](https://github.com/dotnet/docs/issues) to add documentation for the rule in future. Please include all relevant information in the issue to allow the documentation experts to easily author the documentation.
```

**Review by mavasani — APPROVED** [review]

**Pull request merged**